### ISSUE TITLE
Query interpolation issues?

### ISSUE THREAD

**Issue by brettgoulder** [porsager/postgres#549]
I am having trouble with string interpolation.

For some reason this results in an error (`PostgresError: syntax error at or near "$1"`):

```
const query = `SELECT 1;`
await sql`${query}`
```

while this works properly:
```
await sql`SELECT 1;`
```
What am I doing wrong?

**Comment by porsager** [comment]
Does this help? 

https://github.com/porsager/postgres#queries

**Comment by brettgoulder** [comment]
@porsager not really, what am I missing here?

**Comment by brettgoulder** [comment]
Basically I'm just trying to write a function like:
```
async function runQuery(query) {
  const results = await sql`${query}`;
  return results;
}

**Comment by porsager** [comment]
That would allow for sql injection, so you'll have to reach for sql.unsafe. Check the docs 😉

**Comment by manoj1492** [comment]
Is there a way to make the query execution dynamic by wrapping it inside try catch inside a function, somewhat like 
`async function executeQuery(connection_object, query){
  try{
    return await connection_object `${query}`;
  }
  catch(err){
    console.log("Error in executing query", error.query);
  }
}`

**Comment by porsager** [comment]
Hi @manoj1492 .. Have you read the documentation for how to use Postgres.js? I find it hard to parse your question or the purpose of the code, so would you mind try to explain it more thoroughly?
